tests/modules/asset_groups/resources/utils.py

```diff
@@ -42,9 +42,273 @@
     'fluke.jpg': '0b546f813ec9631ce5c9b1dd579c623b',
 }
 
+###################################################################################################################
+# Simple helpers for use with most tests
+###################################################################################################################
+
+
+# Helper for 'normal' tests that don't want to do something clever with the asset group
+def create_simple_asset_group(flask_app_client, user, request, test_root):
+
+    transaction_id, test_filename = tus_utils.prep_tus_dir(test_root)
+    request.addfinalizer(lambda: tus_utils.cleanup_tus_dir(transaction_id))
+
+    data = AssetGroupCreationData(transaction_id, test_filename)
+    response = create_asset_group(flask_app_client, user, data.get())
+
+    json_resp = response.json
+    group_guid = json_resp['guid']
+    asset_guids = [asset['guid'] for asset in json_resp['assets']]
+    ags_guids = [ags['guid'] for ags in json_resp['asset_group_sightings']]
+
+    assert len(ags_guids) == 1
+    assert len(asset_guids) == 1
+    request.addfinalizer(lambda: delete_asset_group(flask_app_client, user, group_guid))
+    return group_guid, ags_guids[0], asset_guids[0]
+
+
+# Helper as used across multiple tests
```

Comment:
rest of the "useful" ones just moved to the top of the file which is where most people will be looking